src/chrome/content/rules/RWTH-Aachen.xml

```diff
@@ -0,0 +1,145 @@
+<ruleset name="RWTH Aachen (partial)">
+	<target host="*.campus.rwth-aachen.de" />
+	<target host="*.embedded.rwth-aachen.de" />
+	<target host="*.eonerc.rwth-aachen.de" />
+	<target host="*.fsmpi.rwth-aachen.de" />
+	<target host="*.itc.rwth-aachen.de" />
+	<target host="*.physik.rwth-aachen.de" />
+	<target host="*.rz.rwth-aachen.de" />

```

Comment:
Our style guide recommends to avoid wildcards.
If they are really needed, please document why and add at least 3 test urls for each wildcard.